Formula/mingw-w64.rb

```diff
@@ -0,0 +1,177 @@
+class MingwW64 < Formula
+  desc "Minimalist GNU for Windows with GCC."
+  homepage "https://mingw-w64.org/"
+  # https://sourceforge.net/p/mingw-w64/wiki2/Cross%20Win32%20and%20Win64%20compiler/
+  url "https://downloads.sourceforge.net/project/mingw-w64/mingw-w64/mingw-w64-release/mingw-w64-v5.0.1.tar.bz2"
+  sha256 "9bb5cd7df78817377841a63555e73596dc0af4acbb71b09bd48de7cf24aeadd2"
+
+  option "with-multilib", "Compile x86_64 compiler without multilib support"
+
+  depends_on "texinfo" => :build
+  depends_on "mpfr"
+  depends_on "libmpc"
+  depends_on "isl"
+  depends_on "gmp"
+
+  resource "binutils" do
+    url "https://ftpmirror.gnu.org/binutils/binutils-2.27.tar.gz"
+    sha256 "26253bf0f360ceeba1d9ab6965c57c6a48a01a8343382130d1ed47c468a3094f"
+  end
+
+  resource "gcc" do
+    url "https://ftpmirror.gnu.org/gcc/gcc-6.3.0/gcc-6.3.0.tar.bz2"
+    sha256 "f06ae7f3f790fbf0f018f6d40e844451e6bc3b7bc96e128e63b09825c1f8b29f"
+  end
+
+  def target_archs
+    %w[i686-w64-mingw32 x86_64-w64-mingw32].freeze
+  end
+
+  def install
+    target_archs.each do |target_arch|
+      resource("binutils").stage do
+        args = %W[
+          --target=#{target_arch}
+          --prefix=#{prefix}
+          --with-sysroot=#{prefix}
+        ]
+        if target_arch.start_with?("i686") || (target_arch.start_with?("x86_64") && build.without?("multilib"))
+          args << "--enable-targets=#{target_arch}" << "--disable-multilib"
+        elsif target_arch.start_with?("x86_64") && build.with?("multilib")
+          args << "--enable-targets=#{target_archs.join(",")}"
+        end
+        mkdir "build-#{target_arch}" do
+          system "../configure", *args
+          system "make"
+          system "make", "install"
+        end
+        info.rmtree
+        (share/"locale").rmtree
+      end
+      ENV.prepend_path "PATH", bin.to_s
+
+      args = %W[
+        --host=#{target_arch}
+        --prefix=#{prefix}/#{target_arch}
+      ]
+      args << "--disable-multilib" if target_arch.start_with?("i686") || (target_arch.start_with?("x86_64") && build.without?("multilib"))
+
+      mkdir "mingw-w64-headers/build-#{target_arch}" do
+        system "../configure", *args
+        system "make"
+        system "make", "install"
+      end
+
+      ln_s "#{prefix}/#{target_arch}", "#{prefix}/mingw"
+
+      resource("gcc").stage buildpath/"gcc"
```

Comment:
(No stupid questions! sometimes I'm not sure about that stuff, and I'm listed as GCC maintainer, so…)

I don't think so… cross-compilation is really quite different from bootstrapped compiler.